src/components/Card/Card.js

```diff
@@ -5,7 +5,7 @@ export default function Card({ name, image }) {
   return (
     <CardContainer>
       <H2>{name}</H2>
-      <IMG src={image} alt="" width="100" heigh="100" />
+      <IMG src={image} alt="" width="100" height="100" />
```

Comment:
maybe change the styling for the h2 and img to:
```suggestion
      const CardContainer = styled.section`
         background-color: whitesmoke;
         text-align: center;
         border-radius: 20px;
         width: 200px;
         box-shadow: 3px 3px 3px #eee;
      h2 {
         font-size: 18px;
      }
      img {  
         border-radius: 20px;
      }
```

but this is just my personal preference :)